--- Serializer.php	2017-02-19 00:23:53.374759977 +0000
+++ /usr/share/pear/XML/Serializer.php	2017-02-19 00:20:42.443400312 +0000
@@ -1212,12 +1212,13 @@
             // but attributes need to be encoded anyways
             // (done here because the rest of the code assumes the same encoding
             // can be used both for attributes and content)
-            foreach ($tag['attributes'] as $k => $v) {
-                $v = XML_Util::replaceEntities($v,
-                    $this->options[XML_SERIALIZER_OPTION_ENTITIES]);
-
-                $tag['attributes'][$k] = $v;
-            }
+// FIXME: hack by gchudov
+//            foreach ($tag['attributes'] as $k => $v) {
+//                $v = XML_Util::replaceEntities($v,
+//                    $this->options[XML_SERIALIZER_OPTION_ENTITIES]);
+//
+//                $tag['attributes'][$k] = $v;
+//            }
         }
         if (is_scalar($tag['content']) || is_null($tag['content'])) {
             if ($this->options[XML_SERIALIZER_OPTION_ENCODE_FUNC]) {
